pages/hold/request/[id].tsx

```diff
@@ -71,13 +75,18 @@ export default function HoldRequestPage({
 
   const handleSubmit = async (e) => {
     e.preventDefault()
-
     try {
       setFormPosting(true)
+      const { patronId, source, pickupLocation } = e.target
+
       const response = await fetch(`${BASE_URL}/api/hold/request/${holdId}`, {
         method: "POST",
-        // TODO: serialize form data
-        body: "",
+        body: JSON.stringify({
+          patronId: patronId.value,
+          source: source.value,
+          pickupLocation: pickupLocation.value,
+          jsEnabled: true,
```

Comment:
I don't agree with this approach but I understand why it's set. It might be the case that you need two endpoints, one for the ajax request and one for the static `<form action="...." >` request (no-js).

If this is true, you can try to reuse the internal functions/logic in `/pages/api/hold/request/[id].ts`. This way, you also don't have to read `jsEnabled` from the request body.